1.16.5/src/main/java/io/github/axolotlclient/mixin/PlayerListHudMixin.java

```diff
@@ -150,4 +165,156 @@ public abstract class PlayerListHudMixin {
 	private boolean axolotlclient$alwaysShowHeadLayer(PlayerEntity instance, PlayerModelPart modelPart) {
 		return Tablist.getInstance().alwaysShowHeadLayer.get() || instance.isPartVisible(modelPart);
 	}
+
+	@Inject(
+		method = "render",
+		at = @At(
+			value = "INVOKE",
+			target = "Lnet/minecraft/client/gui/hud/PlayerListHud;renderLatencyIcon(Lnet/minecraft/client/util/math/MatrixStack;IIILnet/minecraft/client/network/PlayerListEntry;)V"
+		),
+		locals = LocalCapture.CAPTURE_FAILHARD
+	)
+	public void axolotlclient$renderWithoutObjective(
+		MatrixStack matrixStack, int argY, Scoreboard scoreboard, ScoreboardObjective scoreboardObjective, CallbackInfo ci,
+		ClientPlayNetworkHandler clientPlayNetworkHandler, List list, int i, int j, int l, int m, int k,
+		boolean bl, int n, int o, int p, int q, int r, List list2, int t, int u, int s, int v, int y, int z, PlayerListEntry playerListEntry2
+	) {
+		if (!BedwarsMod.getInstance().isEnabled() || !BedwarsMod.getInstance().isWaiting()) {
+			return;
+		}
+		int startX = v + i + 1;
+		int endX = startX + n;
+		String render;
+		try {
+			if (playerListEntry2.getProfile().getName().contains(Formatting.OBFUSCATED.toString())) {
+				return;
+			}
+
+			render = String.valueOf(HypixelAbstractionLayer.getPlayerLevel(playerListEntry2
```

Comment:
This should probably be configurable and depending on option display the star icon.

Reply:
What do you mean exactly?

Reply:
Ability for the user to change what stat it displays. Also, if it displays stars (bedwars level) I think a star icon would make it easier to tell what it is. Maybe even have basic prestige colors, but that's not necessary.

Reply:
Should it just use the already existing option from LevelHead or should it have its own option to set the mode?

Reply:
I think own option just for Bedwars. Just so that you can better see stars or like winstreaks if you want